libraries/AP_HAL/utility/RingBuffer.h

```diff
@@ -81,6 +81,12 @@ class ByteBuffer {
         uint32_t len;
     };
     int peekiovec(IoVec vec[2], uint32_t len);
```

Comment:
I do prefer to have output arg in the end of the method signature, but dunno if this is common in ardupilot. @lucasdemarchi ?